crates/sallyport/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need this still?

Reply:
but the other subcrates still have it, too